lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -210,6 +210,13 @@ negotiate_auto_portchannel:
   set_value: "<state> negotiate auto"
   default_value: true
 
+private_vlan_mapping:
+  _exclude: [ios_xr, N8k]
```

Comment:
Are these new properties really not supported on n8k or are you excluding because they still need to be tested on n8k?